packages/build/src/core/config.js

```diff
@@ -65,11 +68,13 @@ const tLoadConfig = async function ({
   const envValues = mapObj(env, (key, { value }) => [key, value])
   const childEnv = getChildEnv({ envOpt, env: envValues })
   const { packageJson } = await getPackageJson(buildDir)
+  const userNodeVersion = await getUserNodeVersion(nodePath)
```

Comment:
Small performance improvement: should we use `Promise.all()` to run `getPackageJson()` and `getUserNodeVersion()` in parallel?

Reply:
Addressed in https://github.com/netlify/build/pull/2649/commits/639ec6f104f21bb3373aef44ab25b597e085d84e 👍